pointgrey_camera_driver/src/nodelet.cpp

```diff
@@ -350,10 +350,7 @@ class PointGreyCameraNodelet: public nodelet::Nodelet
         wfov_image->info = *ci_;
 
         // Publish the full message
-        if(pub_->getPublisher().getNumSubscribers() > 0)
-        {
-          pub_->publish(wfov_image);
-        }
+        pub_->publish(wfov_image);
 
         // Publish the message using standard image transport
         if(it_pub_.getNumSubscribers() > 0)
```

Comment:
It can be removed. The only overhead is creation of the ImagePtr object. If that's fine, I can update my pull request.